src/background.c

```diff
@@ -854,7 +854,7 @@ validate_commandline (const char *key,
                       GError **error)
 {
   gsize length;
-  const char **strv = g_variant_get_strv (value, &length);
+  g_autofree const char **strv = g_variant_get_strv (value, &length);
```

Comment:
`g_autofree const` sounds like a copy-paste error.

Reply:
This confused me to no end too, but it is not. What happens is that `g_variant_get_strv` is `transfer container`, it's an owned array of unowned strings. The function signature is `const char **` but it must be freed! I didn't know that until reading the documentation.

Reply:
The commit message says "use dup_strv()" but you didn't. Either the commit message needs to be fixed, or the commit.